### PR TITLE
Implement internshipLanguagesValidator and remove 1 rule in Evaluation

### PR DESCRIPTION
**Pull request by Xodoc**

### CODE REVIEW

File: InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        private readonly InternshipLanguagesFixture _languagesFixture;
+
+        public InternshipLanguagesValidationTest()
+        {
+            _languagesFixture = new InternshipLanguagesFixture();
+        }
+
+        [Fact]
+        public async void InternshipLanguagesValidator_IfIncorrectData_ThenThrowsException()
```
Comment (romanstat) [inline InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs]: ThenThrowsException? - no exception is thrown

File: InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+            // act
+            var result = await validator.ValidateAsync(internshipLanguages[1]);
+
+            // assert
+            Assert.False(result.IsValid);
+        }
+              
+        [Fact]
+        public async void InternshipLanguagesValidator_IfCorrectData_ThenThrowsException()
```
Comment (romanstat) [inline InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs]: same

File: InternshipBE/WebApi/Validators/InternshipLanguagesValidator.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+namespace WebApi.Validators
+{
+    public class InternshipLanguagesValidator : AbstractValidator<InternshipLanguage>
+    {
+        public InternshipLanguagesValidator()
+        {
+            RuleFor(languages => languages.InternshipId).NotEqual(0);
+            RuleFor(languages => languages.LanguageType).IsInEnum();
+        }
+    }
```
Comment (romanstat) [inline InternshipBE/WebApi/Validators/InternshipLanguagesValidator.cs]: you added new validator, nothing will break?

File: InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        private readonly InternshipLanguagesFixture _languagesFixture;
+
+        public InternshipLanguagesValidationTest()
+        {
+            _languagesFixture = new InternshipLanguagesFixture();
+        }
+
+        [Fact]
+        public async void InternshipLanguagesValidator_IfIncorrectData_ThenAssertFlase()
```
Comment (romanstat) [inline InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs]: validation failed

File: InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+            // act
+            var result = await validator.ValidateAsync(internshipLanguages[1]);
+
+            // assert
+            Assert.False(result.IsValid);
+        }
+              
+        [Fact]
+        public async void InternshipLanguagesValidator_IfCorrectData_ThenAssertTrue()
```
Comment (romanstat) [inline InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs]: validation passed

**Review by romanstat — APPROVED** [review]

File: InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs [hunk 6]
```diff
@@ -26,7 +26,7 @@ public async void InternshipLanguagesValidator_IfIncorrectData_ThenThrowsExcepti
             // assert
             Assert.False(result.IsValid);
         }
-               // I know DRY (for Roma) Ento vso Zeny. Ego Bei
```
Comment (Evgeniy-Sinicin) [inline InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs]: ### **What's it? 😂🍌🦧**

**Review by Evgeniy-Sinicin — APPROVED** [review]

File: InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs (hunk 6 above)
Comment (Xodoc) [reply InternshipBE/Tests/Validators/InternshipLanguagesValidationTest.cs]: Not well, this is a ban 🗿

**Review by ihhyy — APPROVED** [review]

**Pull request merged**